orchagent/routeorch.h

```diff
@@ -248,6 +364,32 @@ class RouteOrch : public Orch, public Subject
     EntityBulker<sai_mpls_api_t>            gLabelRouteBulker;
     ObjectBulker<sai_next_hop_group_api_t>  gNextHopGroupMemberBulker;
 
+    std::shared_ptr<DBConnector> mAsicDb;
```

Comment:
This is significant changes to routeorch and prefer to have this handled in routeflowcounter.cpp. Let routeorch make some simple calls during add delete operation and rest be handled outside of this file. Please refer to examples of fgnhg or cbf.